app/src/main/java/de/westnordost/streetcomplete/quests/summit_register/AddSummitRegister.kt

```diff
@@ -45,21 +45,25 @@ class AddSummitRegister : OsmElementQuestType<Boolean> {
         val peaks = mapData.nodes.filter { filter.matches(it) }
         if (peaks.isEmpty()) return emptyList()
 
-        val hikingRoutes = mapData.relations
-            .filter { it.tags["route"] == "hiking" }
-            .mapNotNull { mapData.getRelationGeometry(it.id) as? ElementPolylinesGeometry }
-        if (hikingRoutes.isEmpty()) return emptyList()
+        val hikingPaths = mapData.ways.filter { hikingPathsFilter.matches(it) }
+            .mapNotNull { mapData.getWayGeometry(it.id) as? ElementPolylinesGeometry }
 
         // yes, this is very inefficient, however, peaks are very rare
         return peaks.filter { peak ->
-            hikingRoutes.any { hikingRoute ->
-                hikingRoute.polylines.any { ways ->
+            hikingPaths.any { hikingPath ->
+                hikingPath.polylines.any { ways ->
                     peak.position.distanceToArcs(ways) <= 10
                 }
             }
         }
     }
 
+    private val hikingPathsFilter by lazy { """
+        ways with
+          highway = path
+          sac_scale ~ mountain_hiking|demanding_mountain_hiking|alpine_hiking|demanding_alpine_hiking|difficult_alpine_hiking
```

Comment:
```suggestion
        ways with
          highway = path
          and sac_scale ~ mountain_hiking|demanding_mountain_hiking|alpine_hiking|demanding_alpine_hiking|difficult_alpine_hiking
```

or maybe include `route=hiking` (for future use):

```suggestion
        ways with
          (
            highway = path
            and sac_scale ~ mountain_hiking|demanding_mountain_hiking|alpine_hiking|demanding_alpine_hiking|difficult_alpine_hiking
          ) or route = hiking
```

Reply:
`or route = hiking` should be checked on relation - not on ways

Reply:
> So nodes with summit:cross = yes or summit:register will still not be included if they are not close to an included way?

I removed always asking on nodes with `summit:cross` because the only comment on this seemed to not be in favor of using it:

> I am pretty sure that some crosses are on summits inaccessible by regular hikers

Reply:
So the quest should be asked if either a hiking route or a path with appropriate `sac_scale` is nearby?